features/proxy-fields.feature

```diff
@@ -0,0 +1,225 @@
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | <gevraagd aan proxy>            |
+      Dan zoekt de proxy de persoon bij RvIG met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | <vragen aan RvIG>               |
+
+      Voorbeelden:
+      | gevraagd aan proxy | vragen aan RvIG                        |
+      | aanschrijfwijze    | naam,geslachtsaanduiding,partners.naam |
+      | naam               | naam,geslachtsaanduiding,partners.naam |
+      | verblijfplaats     | verblijfplaats                         |
+      | leeftijd,geboorte  | geboorte                               |
+      | nationaliteiten    | nationaliteiten                        |
+      | partners.naam      | partners.naam                          |
+
+    @proxy
+    Abstract Scenario: met fields afgeleid gegeven vragen met of zonder pad <gevraagd aan proxy>
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | <gevraagd aan proxy>            |
+      Dan zoekt de proxy de persoon bij RvIG met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | <vragen aan RvIG>               |
+
+      Voorbeelden:
+      | gevraagd aan proxy                                 | vragen aan RvIG                                   |
+      | nationaliteiten.type,nationaliteiten.nationaliteit | nationaliteit,aanduidingBijzonderNederlanderschap |
+      | nationaliteiten.type,nationaliteit                 | nationaliteit,aanduidingBijzonderNederlanderschap |
+      | overlijden.indicatieOverleden                      | overlijden.datum                                  |
+      | indicatieOverleden                                 | overlijden.datum                                  |
+      | volledige naam                                     | naam                                              |
+      | naam.volledigeNaam                                 | naam                                              |
+
+  Rule: wanneer een gegeven van een ouder, kind of partner wordt gevraagd, moet ook de geslachtsnaam worden gevraagd om het type te bepalen
+    # deze is nodig om het type te bepalen (Ouder, OnbekendOuder, Partner, OnbekendPartner, Kind, OnbekendKind)
```

Comment:
Moet deze nog aangepast worden n.a.v.de reactie vanFrans Rijkers dat we er niet op kunnen vertrouwen dat gegevens conform LO GBA worden aangeboden ?

Reply:
was denk ik ook kwestie van push die nog niet gedaan was